src/SprykerEco/Zed/Easycredit/Persistence/Propel/Schema/spy_payment_easycredit.schema.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\Easycredit\Persistence" namespaceAutoPackage="false" package="src.Orm.Zed.Easycredit.Persistence">
+
+    <table name="spy_payment_easycredit_api_log">
+        <column name="id_payment_easycredit_api_log" required="true" type="INTEGER" primaryKey="true" autoIncrement="true"/>
+
+        <column name="type" required="true" type="VARCHAR"/>
+        <column name="request" required="true" type="VARCHAR"/>
```

Comment:
Shouldn't this be LONGVARCHAR? I guess it's ok with postgresql, but we should have it this way for mysql